lib/commons/dom/is-current-page-link.js

```diff
@@ -12,8 +12,9 @@ const angularRouterLinkRegex = /^#[!/]/;
  * @return {Boolean|null}
  */
 export default function isCurrentPageLink(anchor) {
-  // jsdom can have window.location.origin set to null
-  if (!window?.location.origin) {
+  // jsdom can have window.location.origin set to "null" (the string)
+  // if the url option is not set when parsing the dom string
+  if (window.location.origin === 'null') {
```

Comment:
This doesn't look very robust. How about checking for a protocol?

```suggestion
  if (typeof window.location?.origin !== 'string' || window.location.origin.indexOf('://') === -1) {
```